javascript/utils.js

```diff
@@ -114,6 +114,29 @@ function handleErrors (response) {
   return response
 }
 
+// A proxy method to wrap a fetch call in error handling
+//
+// * url - the URL to fetch
+// * additionalHeaders - an object of additional headers passed to fetch
+//
+async function graciouslyFetch (url, additionalHeaders) {
```

Comment:
Just curious, why "graciously"?

Why not `recursiveFetch` or `deepFetch` or `fetchWithPrejudice`?

Reply:
because we handle errors graciously. It's neither recursive nor deep, it just wraps `fetch` complete with error handling.